src/osd/OSD.cc

```diff
@@ -4510,6 +4519,15 @@ void OSD::tick()
   do_waiters();
 
   tick_timer.add_event_after(OSD_TICK_INTERVAL, new C_Tick(this));
+
+  if (is_active()) {
+    utime_t now = ceph_clock_now();
```

Comment:
it's better to use monoclock here, we met lots of clock backoff causing weird behavior